static/src/javascripts/projects/common/modules/experiments/tests/utils/comment-blocker.js

```diff
@@ -0,0 +1,21 @@
+define([
+    'common/utils/config',
+    'lodash/collections/reduce'
+], function(
+    config,
+    reduce
+) {
+
+    var CommentBlocker = {};
+
+    CommentBlocker.hideComments = function(shortUrlSlug) {
+
+        var sUrlInt = shortUrlSlug !== null ? reduce(shortUrlSlug.split(''), function(sum, ch) {
```

Comment:
@gtrufitt Yeah a played with various fronts for a few hours and it loooked to come out about even ...